docs/UserGuide.md

```diff
@@ -563,8 +564,11 @@ Flagship data is saved automatically as a [JSON](#glossary) file `[JAR file loca
 Advanced users are welcome to update data directly by editing that data file.
 
 <div markdown="span" class="alert alert-danger">
-⚠️ If your changes to the data file make it invalid, such as adding invalid formatting or duplicate internship entries, Flagship will discard all data and start with an empty data file at the next run.
-Hence, it is recommended to take a backup of the file before editing it.
+⚠️ Be extremely careful if you make any changes to the data file. If you make incorrect formatting changes or insert duplicate internship entries,
```

Comment:
Also add the case of invalid values given to fields (ie no duplicates, correct formatting but wrong values

Reply:
Rectified